generators/src/plop-interfaces.ts

```diff
@@ -2,6 +2,7 @@ export interface Answers {
   componentName: string;
   propName: string;
   ruleName: string;
+  referenceRepo: string;
```

Comment:
Should we maybe have this be optional, with it defaulting to pf-react since that is what _most_ of our mods are for?

Reply:
@adamviktora wdyt?

For context there was some discussion on Slack:

- My intent was to have the enum sort of be the source of truth for any repo names we create codemods for, with that being the only thing we would have to update (optimistically at least; the prompt creates an array for the values of this enum). It would involve less manually typing an exact repo correctly/ensuring the repo name is exactly correct for the PR url we create for the files
- Alternatively, the repo name prompt becomes a regular input that defaults to `patternfly-react`, but have the ability to input custom text if you want